homeassistant/components/automation/__init__.py

```diff
@@ -267,19 +264,21 @@ def is_on(self) -> bool:
     @asyncio.coroutine
     def async_added_to_hass(self) -> None:
         """Startup with initial state or previous state."""
-        enable_automation = False
+        enable_automation = DEFAULT_INITIAL_STATE
```

Comment:
I think some debug level logs would be helpful in this routine. It would be good to know where the state was sourced from if you aren't getting the result you expected. It'll be hard to track down otherwise.